DashKitUI/Views/DashSettingsView.swift

```diff
@@ -275,6 +280,32 @@ struct DashSettingsView<Model>: View where Model: DashSettingsViewModelProtocol
             .cancel()
         ])
     }
+
+    var suspendOptionsActionSheet: ActionSheet {
```

Comment:
[nit] Might be worth adding localization comments for the Text throughout the body of this property, although I guess since we don't have a framework-localized version of Text yet it'll have to change in the future anyway

Reply:
Maybe we should just go and do this. It should be trivial.